templates/content/node--oe-list-page--full.html.twig

```diff
@@ -0,0 +1,55 @@
+{#
+/**
+ * @file
+ * Theme override to display a list page node.
+ */
+#}
+{% set display_submitted = false %}
+
+<article{{ attributes }}>
+
+  {{ title_prefix }}
+  {% if not page %}
+    <h2{{ title_attributes }}>
+      <a href="{{ url }}" rel="bookmark">{{ label }}</a>
+    </h2>
+  {% endif %}
+  {{ title_suffix }}
+
+  {% if display_submitted %}
```

Comment:
Not sure we need this chunk since we always set "display_submitted" to false anyways. We could probably just remove it.